spec/identity/hostdata_spec.rb

```diff
@@ -225,7 +225,7 @@
     end
   end
 
-  describe '.s3' do
+  describe 'app_secrets_s3' do
```

Comment:
nit: use dot notation for class methods

```suggestion
  describe '.app_secrets_s3' do
```